packages/caver-core-method/src/index.js

```diff
@@ -459,27 +459,66 @@ const buildSendFunc = (method, isSendTx) => (...args) => {
 
     const isGasPriceInputMissing = isSendTx && _.isObject(payload.params[0]) && payload.params[0].gasPrice === undefined
 
-    // If gasPrice input is missing, call getGasPrice rpc
-    if (!isGasPriceInputMissing) {
+    // The TxTypeEthereumDynamicFee transaction does not use the gasPrice field,
+    // so we need to check `maxPriorityFeePerGas` and `maxFeePerGas` field instead of `gasPrice`.
+    const isDynamicFeeTx = isSendTx && payload.params[0].type === TX_TYPE_STRING.TxTypeEthereumDynamicFee
+    const filledDynamicGasFeeTx =
+        isDynamicFeeTx && payload.params[0].maxPriorityFeePerGas !== undefined && payload.params[0].maxFeePerGas !== undefined
+
+    // If gasPrice is missing, have to fill gasPrice field before sending tx
```

Comment:
```suggestion
    // gasPrice is already set so it is ok to send transaction.
```

Reply:
Updated this comment. And add additional comment below to make the logic more clear. :)